bpf/lib/common.h

```diff
@@ -655,6 +655,7 @@ enum {
 #define DROP_HOST_NOT_READY	-202
 #define DROP_EP_NOT_READY	-203
 #define DROP_NO_EGRESS_IP	-204
+#define DROP_PUNT_PROXY		-205
```

Comment:
When adding a new drop code we should also update [api/v1/flow/flow.pb](https://github.com/cilium/cilium/blob/452457e397cfe1bf80d8f59f3c84aa40bf0785aa/api/v1/flow/flow.proto#L375), files generated from it, and [pkg/monitor/api/drop.go](https://github.com/cilium/cilium/blob/452457e397cfe1bf80d8f59f3c84aa40bf0785aa/pkg/monitor/api/drop.go#L17)

Reply:
thx fixed

Reply:
Would it make sense to add a comment here saying that this is not a real drop reason? We seem to do something similar for `DROP_NAT_NOT_NEEDED`

https://github.com/cilium/cilium/blob/c6f05a5360918e646126d313a20dd9fb1b4db613/bpf/lib/common.h#L626

Reply:
makes sense, done now.